platform/persistence/platform-persistence-core-impl/src/main/java/org/codice/ddf/persistence/internal/PersistentStoreImpl.java

```diff
@@ -99,8 +74,8 @@ public void add(String type, Map<String, Object> properties) throws PersistenceE
         LOGGER.debug("Adding entry of type {}", type);
 
         // Set Solr Core name to type and create/connect to Solr Core
-        SolrClient coreSolrClient = getSolrCore(type);
-        if (coreSolrClient == null) {
+        SolrClient solrClient = getSolrClient(type);
+        if (solrClient == null) {
```

Comment:
Should we log something everywhere `getSolrClient()` fails to return a client? Right now, the only that gets logged is a generic debug message, which won't help figure out what's going on if an update to the persistence store fails.